### ISSUE TITLE
Dropdown lists not accessible with screen readers

### ISSUE THREAD

**Issue by ptorpey** [pinokiocomputer/pinokio#1008]
I am blind and use the screen reader JAWS for Windows.

I don't know how the dropdown list views are coded in Pinokio, but it is not possible to arrow down through the list of items in any of these lists using standard navigation commands with a screen reader.  This applies not only to JAWS, but also to NVDA (an popular open source screen reader), as well as Narrator (built into Windows).

I am using Chrome in Windows 11.

Here are the usual steps taken to work with pulldown lists:
1.  Tab to the list element.
2. 2. Hit enter to activate the list
3. 3. Use the up/down arrows to navigate through the list.
4. 4.  Hit enter again to make the selection
5. 5.  Tab away from the list element.
What happens in Pinokio pulldown menus:
1.  I tab to the list element
2. 2.  I hit enter
3. 3.  Focus moves away from the list element and to the top of the screen
4. 4.  In some list elements, focus may stay on the list, but arrowing down does nothing.  On other list elements, I can arrow down two or three times to "see" other items in the list, but if I try to arrow further, instead of arrowing through the list, I am brought out of the list.  Thus there is no way to choose items deeper than two or three items in the list.
I see this most frequently when trying to choose a model, such as in the Open AI dialog, or selecting to use my GPU or CPU in the Whisper Web UI in the Diraization dialog.

This is a serious design issue that affects the accessibility of Pinockio in general and many of the apps hosted by Pinokio.

Hoepfully this can be addressed by adhering to proper web design guidelines for accessibility.  Devs can test such issues by using Window's built in Narrator, the free NVDA screen reader, or the 40 minutes demo mode of JAWS.

Thanks for your attention to these issues.

--Pete
